libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/ScalesWith.tsx

```diff
@@ -0,0 +1,36 @@
+import { iconInlineProps } from '@genshin-optimizer/common/svgicons'
+import { StatIcon } from '@genshin-optimizer/gi/svgicons'
+import {
+  ArtifactStatWithUnit,
+  OptTargetContext,
+} from '@genshin-optimizer/gi/ui'
+import { Alert } from '@mui/material'
+import { useContext } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+
+export default function ScalesWith() {
+  const { t } = useTranslation('page_character_optimize')
+  const { scalesWith } = useContext(OptTargetContext)
+  return (
+    !!scalesWith.size && (
+      <Alert severity="info" variant="filled">
```

Comment:
Can we use the new standard for alert?
![image](https://github.com/user-attachments/assets/62ffdbf2-a39b-4bb9-b1a8-919d86e89aeb)

Reply:
you meant the `variant="standard"`? I'll give it a try.